numpy/_core/src/multiarray/nditer_api.c

```diff
@@ -2091,509 +2083,164 @@ npyiter_copy_to_buffers(NpyIter *iter, char **prev_dataptrs)
     npyiter_opitflags *op_itflags = NIT_OPITFLAGS(iter);
     NpyIter_BufferData *bufferdata = NIT_BUFFERDATA(iter);
     NpyIter_AxisData *axisdata = NIT_AXISDATA(iter),
-                    *reduce_outeraxisdata = NULL;
+                     *outer_axisdata = NULL;
```

Comment:
If you do make further changes, might as well make this its own definition (as you've done elsewhere; is clearer).